DependencyInjection/Configuration.php

```diff
@@ -19,8 +19,10 @@ public function getConfigTreeBuilder()
         $treeBuilder
             ->getRootNode()
             ->children()
-            ->scalarNode('default_class_namespace')->setDeprecated()->end()
-            ->arrayNode('mappings')
+            ->scalarNode('default_class_namespace')->end()
+            ->scalarNode('api_root_url')->setDeprecated()->end()
```

Comment:
I can be removed, but I think it would be better to leave it in as deprecated for a version or two, so that developers get warnings to change the config.

Reply:
`api_root_url` is introduced in this branch, so there's no BC to worry about

Reply:
Ah, my mistake. I've removed it, it's not used.